### ISSUE TITLE
Organize Wiki Sidebar and Home

### ISSUE THREAD

**Issue by niyaziulke** [bounswe/2021SpringGroup9#234]
Our Wiki needs some gardening. The tasks are:

- [x] Show that Kadir Kalkan is no longer an active member of the group.
- [x] Group design diagrams under Design title.
- [x] Write project name to home.
- [x] Solve any other problems related to the wiki organization. (The changes must be stated.)

**Deadline: 31.10.2021

**Comment by niyaziulke** [comment]
* I wrote project name to the home page and created a new section named "About Postory Project" to provide a longer explanation. 
*  "Project" subtitle is renamed as "Project Documentation".
* Milestone Reports and Project Plans are now grouped under the same subtitle because the reports are actually evaluations of our achievements in comparison to the plans.
* Practice app documentation is now under another subtitle.
* I removed "Us" part from the sidebar, because we do not need to access our personal information pages frequently.  It is better to keep the sidebar shorter for convenience. There are still links at the home page.
* "Diagrams" are renamed as "Design"
* Since Kadir Kalkan is sadly no longer a member of the team, I crossed out the name.
* There were anchor texts such as "Milestone Report 1" . These are also changed to make it clear that the report is "CMPE 352 Milestone Report 1".
